internal/process/provisioning/create_runtime_resource_step.go

```diff
@@ -58,7 +58,8 @@ func (s *CreateRuntimeResourceStep) Run(operation internal.Operation, log logrus
 		return operation, 0, nil
 	}
 
-	kymaResourceName, kymaResourceNamespace := getKymaNames(operation)
+	kymaResourceName := steps.KymaRuntimeResourceName(operation)
```

Comment:
once, we set the KymaName in the previous step, we can just rely on an operation field

Reply:
Changed in the next commit.